libs/analytics/migrations/202104281912.do.0078-contactType.sql

```diff
@@ -0,0 +1,56 @@
+-- Recreates the view only to add the new column contact_type
+DROP MATERIALIZED VIEW IF EXISTS contactview_metadata CASCADE;
+CREATE MATERIALIZED VIEW contactview_metadata AS
+  SELECT doc->>'_id' AS uuid,
+    doc->>'name' AS name,
+    doc->>'type' AS type,
+    doc->>'contact_type' AS contact_type,       --> only this is new
+    doc#>>'{contact,_id}' AS contact_uuid,
+    doc#>>'{parent,_id}' AS parent_uuid,
+    doc->>'notes' AS notes,
+    TIMESTAMP WITH TIME ZONE 'epoch' + (doc->>'reported_date')::numeric / 1000 * interval '1 second' AS reported
+  FROM raw_contacts;
+
+CREATE UNIQUE INDEX contactview_metadata_uuid ON contactview_metadata (uuid);
+CREATE INDEX contactview_metadata_contact_uuid ON contactview_metadata (contact_uuid);
+CREATE INDEX contactview_metadata_parent_uuid ON contactview_metadata (parent_uuid);
+CREATE INDEX contactview_metadata_type ON contactview_metadata (type);
+
+-- NOTE: The recreation of the view above caused 4 other views to be dropped in cascade,
+--       here are the scripts to recreate them:
+
```

Comment:
I think we should link to the original migration that created each of these (including the line numbers). Something like: https://github.com/medic/medic-couch2pg/blob/master/libs/analytics/migrations/201711071603.do.2635-removeNestedContactReferences.sql#L26

Reply:
Yes better to keep track of that, adding...